compiler/src/dotty/tools/dotc/ast/tpd.scala

```diff
@@ -573,6 +573,8 @@ object tpd extends Trees.Instance[Type] with TypedTreeInfo {
       Try(tree: Tree)(expr, cases, finalizer)
   }
 
+  override def skipTransform(tree: Tree)(implicit ctx: Context) = tree.tpe.isError
```

Comment:
It might be cleaner to just override the `transform` as follows:
```scala
override def transform(tree: Tree)(implicit ctx: Context): Tree = 
  if (tree.tpe.isError) tree else super.transform(tree)
```